### ISSUE TITLE
[BUG] - CustomStatusPresets - 'Quick Select' status aren't usable.

### ISSUE THREAD

**Issue by TheKrush** [mwittrien/BetterDiscordAddons#2768]
**Describe the bug**
Can no longer use the custom status presets. They appear in the settings when added:
![image](https://github.com/user-attachments/assets/d5844d29-2b8d-4699-a267-5a0b71d5b633)

But the option for applying the custom status from the quick select doesn't exist anymore.
![image](https://github.com/user-attachments/assets/317f3be6-e6d3-4289-9090-319aeda957a0)

**To Reproduce**
Make a new custom status and 'Save as Quick Select'

**Expected behavior**
That you should be able to apply 'Quick Select' status'.

**Screenshots**
See above.

**Discord Build (stable, ptb or canary):**
stable

**Additional context**
None.

**Comment by misterpyrrhuloxia** [comment]
I've been having this same issue for at least a few days or even a week (not sure exactly how long)

**Comment by mwittrien** [comment]
Right click the status bubble

**Comment by TheKrush** [comment]
Right-clicking the status bubble doesn't do anything.

![image](https://github.com/user-attachments/assets/4dd0399b-0e7e-4818-9c98-c8696a402b85)
Right-clicked everything in this screenshot and none of them do anything.

**Comment by TheKrush** [comment]
As per #2758 deleting **0BDFDB.data.json** fixed the issue.

![image](https://github.com/user-attachments/assets/29102f3c-d4e3-4c78-8b30-2e147cb2ec9e)

**Comment by TheKrush** [comment]
Might be worth noting that in the pop-up notes.

**Comment by misterpyrrhuloxia** [comment]
@TheKrush, can you please test out saving some statuses with emojis? I'm curious if other people are experiencing the same issue that I am, as detailed [here](https://github.com/mwittrien/BetterDiscordAddons/issues/2758#issuecomment-2311833844) on #2758.  
TL;DR emojis in my saved custom statuses aren't visible in the selector list, but the emojis are still applied just fine when I select any of the statuses.

**Comment by TheKrush** [comment]
@misterpyrrhuloxia works fine, just doesn't show it in the menu when selecting the quick save, but it applies it just fine.

**Comment by misterpyrrhuloxia** [comment]
I've just submitted #2770 for this problem. Thank you for testing that for me, @TheKrush.
